### ISSUE TITLE
expert pattern editor

### ISSUE THREAD

**Issue by williballenthin** [mandiant/flare-floss#755]
consider developing a standalone tool for creating/editing expert pattern databases.

and/or integrate this into the QS ui natively so its easy for users to tweak the tool to their comfort.

**Comment by mr-tz** [comment]
+1 on this via #776

**Comment by c-urly** [comment]
Hi @williballenthin / @mr-tz ,
I believe we want to make all the database editable right? not just the expert pattern database.

Do we want to make a separate tool that will help to tweak the database and after that QS will use the updated database? or we want to integrate the tweaking into QS?

So what's the plan with QS, will it be a command line tool? or GUI tool?

**Comment by mr-tz** [comment]
Yes, ideally all database can be edited, but it's most relevant for the expert data (based on personal/crowd-sourced knowledge).

Some easy to use GUI tool would likely be best, so it may be separate or embedded in QS. Some functionality of QS is CLI-focused but quite a bit would benefit from a real GUI.

**Comment by mr-tz** [comment]
Overall, it's not a high priority but it would be a high effort, high reward feature for sure.
